### PR TITLE
[#59] Bump to relude-0.4.0

### LINKED ISSUE #59: Bump up to relude-0.4.0

### PR DESCRIPTION
**Pull request by nixorn**
close #59

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -15,7 +15,10 @@ The change log is available [on GitHub][2].
 
 * [#49](https://github.com/kowainik/life-sync/issues/49):
   Core module with internal types.
-  
+
+* [#59](https://github.com/kowainik/life-sync/issues/59):
+  Bump to relude-0.4.0.
```
Comment (vrom911) [inline CHANGELOG.md]: You can also mention, you upgraded to GHC-8.4.4 👍

File: life-sync.cabal [hunk 2]
```diff
@@ -13,7 +13,7 @@ stability:           experimental
 build-type:          Simple
 cabal-version:       2.0
 tested-with:         GHC == 8.2.2
-                   , GHC == 8.4.3
+                   , GHC == 8.4.4
```
Comment (vrom911) [inline life-sync.cabal]: I guess, you can update Travis as well to reflect this change:
https://github.com/kowainik/life-sync/blob/7e889f3038a72a365d7fb860158f6a45c297c8ec/.travis.yml#L28-L39

File: .travis.yml [hunk 3]
```diff
@@ -25,18 +25,18 @@ matrix:
         - ghc-8.2.2
         - cabal-install-head
 
-  - ghc: 8.4.3
+  - ghc: 8.4.4
     env: GHCVER='8.4.3' CABALVER='head'
```
Comment (vrom911) [inline .travis.yml]: One small thing left here :)

Comment (nixorn) [reply .travis.yml]: Done)

**Pull request merged**